lib/views/commit-view.js

```diff
@@ -83,7 +75,16 @@ export default class CommitView extends React.Component {
         return;
       }
 
-      const fakeEvent = getFakeKeyDownEvent(keyCode);
+      if (!FakeKeyDownEvent) {
+        FakeKeyDownEvent = class extends CustomEvent {
+          constructor(kCode) {
+            super('keydown');
+            this.keyCode = kCode;
+          }
+        };
+      }
+
+      const fakeEvent = new FakeKeyDownEvent(keyCode);
```

Comment:
/cc @annthurium - took the liberty to 🎨 this. Hope that's okay!

Reply:
awesome, thank you!!